test_runner/regress/test_tenant_size.py

```diff
@@ -532,7 +532,24 @@ def get_current_consistent_size(
     assert size_after == prev, "size after restarting pageserver should not have changed"
 
 
-@xfail_on_postgres(PgVersion.V15, reason="Test significantly more flaky on Postgres 15")
+def assert_approx_equal(size_a, size_b):
```

Comment:
I would mention size in the method name, something like

```suggestion
def assert_size_approx_equal(size_a, size_b):
```

or 

```suggestion
def assert_approx_size_equal(size_a, size_b):
```

Reply:
Changed out of band